encoding/yaml/parse_test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```

Comment:
Those tests files are not covering all the features of the lib.
Could you port those from here: https://github.com/nodeca/js-yaml/tree/master/test ?

Reply:
JS files copied. Will do the conversion in ts/deno asap.

Reply:
Will also be included issues specific tests from js-yaml repository.